lib/fixture_factory/registry.rb

```diff
@@ -46,12 +46,12 @@ module Registry
       #   factory(:admin, via: :users, like: :bob_admin, class: 'User')
       # end
       def factory(name, options = {}, &block)
-        if options.key?(:class)
+        if options.key?(:class_name)
           ActiveSupport::Deprecation.warn(<<~MSG.squish)
-            factory class: option is deprecated and will be removed.
-            Please use the class_name: option instead.
+            factory class_name: option is deprecated and will be removed.
```

Comment:
It will be annoying for the users to deprecate this option as well. Can we just support both for now but prefer `class` in the documentation?

Reply:
I think that makes sense. That way we don’t have to update existing applications unless we want to reference the constant directly.